README.org

```diff
@@ -20,6 +19,11 @@ is known.  You can also add an entry for *yourself* if you don't accept
 any donations but would like users to donate to charity instead (make
 that explicit).
 
+[fn:1] That's actually quite time consuming, which is why we need this
+list.
+
+[fn:2] Don't be shy.
```

Comment:
It appears that the Org formatter that is being used by Github doesn't understand this format either. That's probably why I used the format that I originally used. While not linkified, at least it's not noisy.

Reply:
Yeah, I wish the org-ruby formatter would be updated and improved.